Tasks/SshV0/ssh2helpers.ts

```diff
@@ -147,7 +147,11 @@ export function runCommandOnRemoteMachine(
             }).on('data', (data) => {
                 if (data) {
                     // "data" can be a buffer. Format it here so it outputs as a string
-                    console.log(data.toString('utf8'));
+                    if (tl.getPipelineFeature("redirectTaskOutputToProcessStdout")) {
```

Comment:
nit: please add the task name to the flag name for clarity

Reply:
do you mean that this flag will be reused in multiple tasks? In this case, it's ok for me to keep the current name 🙂

Reply:
I have found a few more [buffer conversions](https://github.com/search?q=repo%3Amicrosoft%2Fazure-pipelines-tasks+%22data.toString%28%27utf8%27%29%22+path%3A%2F%5ETasks%5C%2F%2F&type=code).
The solution might also be applicable in these scenarios.